pygmt/src/blockm.py

```diff
@@ -111,6 +112,12 @@ def blockmean(data=None, x=None, y=None, z=None, outfile=None, **kwargs):
 
     {I}
 
+    restype: str
+        Result type calculated by blockmean. Use **n** to report the number of
```

Comment:
```suggestion
        [**m**\|\ **n**\|\ **s**\|\ **w**].
        Result type calculated by blockmean. Use **n** to report the number of
```

I think this should have the potential arguments for `restype`